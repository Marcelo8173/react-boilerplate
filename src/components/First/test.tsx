import React from 'react'
import { render, screen } from '@testing-library/react'
import { First } from '.'

describe('<First />', () => {
  it('should render the heading', () => {
    //renderizando o componet
    const { container } = render(<First />)

    expect(
      screen.getByRole('heading', { name: /realize b2c/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
