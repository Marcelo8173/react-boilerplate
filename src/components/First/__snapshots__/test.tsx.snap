// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<First /> should render the heading 1`] = `
<main>
  <h1>
    realize b2c
  </h1>
</main>
`;
